scss/_regions/_footer.scss

```diff
@@ -102,7 +102,8 @@
       }
 
       .social-icon {
-        margin: ($base-spacing / 2);
+        margin: 8px;
+        padding: 1px 1px 0;
```

Comment:
Sure thing. Defs a magic number, but part of the magic is that it depends on a max specified number of social media icons being present. We've added an extra one and that causes stuff to break. This all in general feels more like it shouldn't be part of a pattern library that should be agnostic to the number of social media icons added. That feels more specific to the theme. I can add this as part of the comment for future refs.